app/site/controllers.py

```diff
@@ -87,6 +80,10 @@ def datapackage_show(publisher, package):
       200:
         description: Succesfuly loaded
     """
+    user, exception = get_user_from_cookie()
```

Comment:
We should not be repeating the same code over and over again in different locations! DRY!

If what you want here is to set the logged user (if there is one) before every request this is a very common pattern and should be done in one global location. Please do a quick google or see http://stackoverflow.com/questions/19531721/flask-g-user-and-before-request